File: sum.c
#include<stdint.h>

int main (){
  int a = 0, b = 0 , sum= 0;
  printf("Enter two numbers");
  scanf ("%d%d" ,&a, &b);
  sum = a + b;
  printf( "The sum of: %d and: %d is: %d ", a,b,sum);
}

File: Learn_Day7/LearnJava/BasicJava/src/sum.java
import java.util.Scanner;

public class sum{

    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter 2 Numbers \n");
        int num1, num2, sum;

        num1=sc.nextInt();
        num2=sc.nextInt();
        sum = num1+num2;
        System.out.println("Result :   "+sum);
    }


}